### PR TITLE
Issue 64: Use mailgun to send email when user volunteers for a project

### PR DESCRIPTION
**Pull request by FangJennyYuan**
Integrated with mail gun to send emails. 

issue: https://github.com/SeattleVoluntech/seattlevoluntech/issues/64
pr: https://github.com/SeattleVoluntech/seattlevoluntech/pull/105

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -10,3 +10,7 @@ spring.datasource.password=${DB_PASS}
 
 #Do nothing
 spring.jpa.hibernate.ddl-auto=none
+
+#Mailgun email sender properties:
+mailgun.domain: sandbox08786ed216874ad697286dcf97357a43.mailgun.org
+mailgun.api.key: 41a98182603ad234d941c01860d8e426-c8c889c9-270b8e5a
```
Comment (Richard-outreach) [inline src/main/resources/application.properties]: Put these as environment variables. The repo is public and you probably want to generate a new one now.

Comment (FangJennyYuan) [reply src/main/resources/application.properties]: ahh good point, removing

Comment (FangJennyYuan) [reply src/main/resources/application.properties]: Rotated key, and moved those variables into environment variables

Comment (Seiyaroo) [reply src/main/resources/application.properties]: Checks are passed @Richard-outreach up to you if you wanna push it through. Pinging for notice.

**Pull request merged**